.github/CODEOWNERS

```diff
@@ -73,3 +73,5 @@
 /src/connectedmachine/ @farehar
 
 /src/ip-group/ @haroldrandom
+
+/src/storagesync/ @jsntcy
```

Comment:
Shall we use storage-sync as the extension name?

Reply:
Not sure which one should be as both patterns exist in CLI, and we can have a discussion.

---
In reply to: [392828794](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392828794) [](ancestors = 392828794)

Reply:
There is guideline on [command naming](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior). But we can discuss it for sure.

Reply:
az [ group ] [ subgroup ] [ command ] {parameters}
Multi-word subgroups should be hyphenated (e.g. foo-resource instead of fooresource)
- It seems storagesync is group, but not subgroup.
- Also discuss within our team, they agree I can keep it without hyphen.
 So I'll keep it without hyphen.

---
In reply to: [392864381](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392864381) [](ancestors = 392864381)